WISE-Community/WISE
# Graph Component: Multiple graphs can now be imported into a single graph. Also added some tests.

How to test
1. Create a step with Graph 1
2. Create a step with Graph 2
3. Create a step with Graph 3 that will import Graph 1 and Graph 2
4. Create a step with Graph 4 that will import Graph 1 and show Graph 2
5. Preview the project to test the steps
6. Make sure Graph 3 imports Graph 1 and Graph 2 and both lines can be edited
7. Make sure Graph 4 imports Graph 1 and shows Graph 2. Make sure Graph 1 can be edited and Graph 2 can not be edited.

Test Jenn's Thermo grid/model/graph.

#1883
